mxnet-predict.spec

```diff
@@ -17,6 +21,8 @@ rm -rf ../build; mkdir ../build; cd ../build
 # use LAPACK functions in OpenBLAS:
 # manually set MXNET_USE_LAPACK=1 and turn off USE_LAPACK in cmake
 export CFLAGS="-DMXNET_USE_LAPACK=1 -DMXNET_THREAD_LOCAL_ENGINE=1"
+export PYTHONV=$(echo $PYTHON_VERSION | cut -f1,2 -d.)
+export PYTHON3V=$(echo $PYTHON3_VERSION | cut -f1,2 -d.)
 
 cmake ../%{n}-%{realversion} -GNinja \
     -DCMAKE_CUDA_COMPILER_LAUNCHER=ccache \
```

Comment:
@mrodozov , we also need to clean this up, we should use our compiler

Reply:
ok let me see about that

Reply:
I'm trying to understand what is ccache doing as a "compiler launcher". we don't use cuda with mxnet so I'll guess I have to change it to gcc (I did it and it's building), but also because we don't use cuda do we need this flag ?

Reply:
just drop ccache dependency and remove the 3 LAUNCHER lines from cmake command and you should be fine.